readme_4062.txt

```diff
@@ -0,0 +1,37 @@
+The objective of this Pull Request is to make installation more robust on mac's regarding freetype paths   
```

Comment:
This is useful information, but should not go into the repo as a tracked file.  This seems better suited as a commit message or comments in the code.